src/runtime/policy/ingress-validation.ts

```diff
@@ -288,6 +288,14 @@ export class IngressValidation {
       case 'TypeVariable': {
         const typeVar = (type as TypeVariable).variable;
         const canReadSubset = type.canReadSubset;
+        if (type.isResolved()) {
```

Comment:
nit - i think the line above:
`const canReadSubset = type.canReadSubset;`
can be moved to after the new `if` block.

Reply:
done.